src/mlpack/tests/callback_test.cpp

```diff
@@ -135,4 +138,32 @@ BOOST_AUTO_TEST_CASE(LRWithOptimizerCallback)
     BOOST_REQUIRE_GT(stream.str().length(), 0);
 }
 
+/**
+ * Test a Linear SVM model with EndOptimization callback.
+ */
+BOOST_AUTO_TEST_CASE(LinearSVMCallbackTest)
+{
+  const size_t numClasses = 2;
+  const double lambda = 0.0001;
+  const double delta = 1.0;
+
+  // A very simple fake dataset
+  arma::mat dataset = "2 0 0;"
+                      "0 0 0;"
+                      "0 2 1;"
+                      "1 0 2;"
+                      "0 1 0";
+
+  //  Corresponding labels
```

Comment:
See comment above, also looks like you used two spaces after `//`, but only one is needed; sometimes I'm very picky about the style.